sdk/textanalytics/ai-text-analytics/review/ai-text-analytics.api.md

```diff
@@ -149,6 +155,8 @@ export interface MinedOpinion {
 export interface OpinionSentiment {
     confidenceScores: SentimentConfidenceScores;
     isNegated: boolean;
+    length: number;
+    offset: number;
     sentiment: DocumentSentimentLabel;
     text: string;
 }
```

Comment:
SentenceSentiment (L205) also had offset/length properties that need to be restored IIRC.

Reply:
Done.